go-crud/books/controllers.go

```diff
@@ -0,0 +1,257 @@
+package books
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+)
+
+var data []book
+
+func loadBookFile() error {
+	_, err := os.Stat("books.json")
```

Comment:
You are using `"books.json"` the data store path more than once, it's better to store it in a variable or even better to get it as a parameter to this function. It would make this method more testable.